inputremapper/injection/mapping_handlers/abs_to_abs_handler.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+# input-remapper - GUI for device specific keyboard mappings
+# Copyright (C) 2022 sezanzeb <proxima@sezanzeb.de>
+#
+# This file is part of input-remapper.
+#
+# input-remapper is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# input-remapper is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with input-remapper.  If not, see <https://www.gnu.org/licenses/>.
+from typing import Tuple, Optional, Dict
+
+import evdev
+from evdev.ecodes import EV_ABS
+
+from inputremapper import exceptions
+from inputremapper.configs.mapping import Mapping
+from inputremapper.event_combination import EventCombination
+from inputremapper.injection.global_uinputs import global_uinputs
+from inputremapper.injection.mapping_handlers.axis_transform import Transformation
+from inputremapper.injection.mapping_handlers.mapping_handler import (
+    MappingHandler,
+    HandlerEnums,
+    InputEventHandler,
+)
+from inputremapper.input_event import InputEvent, EventActions
+from inputremapper.logger import logger
+
+
+class AbsToAbsHandler(MappingHandler):
+    """Handler which transforms EV_ABS to EV_ABS events"""
+
+    _map_axis: Tuple[int, int]  # the (type, code) of the axis we map
+    _output_axis: Tuple[int, int]  # the (type, code) of the output axis
+    _transform: Optional[Transformation]
+    _target_absinfo: evdev.AbsInfo
+
+    def __init__(
+        self,
+        combination: EventCombination,
+        mapping: Mapping,
+        **_,
+    ) -> None:
+        super(AbsToAbsHandler, self).__init__(combination, mapping)
+
+        # find the input event we are supposed to map
```

Comment:
I think I'd add an example to this. I hope I got this right

```suggestion
        # find the input event we are supposed to map. If the input combination is
        # BTN_A + ABS_X + BTN_B, then use the value of ABS_X for the transformation
```